lib/logger.js

```diff
@@ -75,6 +75,10 @@ Logger.prototype.exit = function (code) {
 	process.stdout.write('');
 	process.stderr.write('');
 
+	if (this.watch) {
```

Comment:
IMO you should place this guard in the `cli` module, the logger shouldn't have to know about watchers.